test/integration/cluster/cluster_test.go

```diff
@@ -97,7 +97,10 @@ var _ = Describe("Cluster-Controller", func() {
 				Spec: *clusterSpec.DeepCopy(),
 			}
 
-			Expect(apiclient.Create(ctx, cluster)).ShouldNot(HaveOccurred())
+			Eventually(func() error {
+				return apiclient.Create(ctx, cluster)
+			}).Should(Succeed())
```

Comment:
Should we add a more appropriate wait condition/sleep to scripts/ci-integration.sh instead?

Reply:
unfortunately there's no real good way to wait for this. The manager/webhook pod doesn't have a good readiness probe. The pod becomes ready ready but the web hooks aren't necessarily ready. The best way I've found to do this is to just try to create until it succeeds.